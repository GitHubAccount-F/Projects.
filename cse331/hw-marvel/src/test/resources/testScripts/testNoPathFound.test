#Tests when there is not a path found

#Creates graphs
CreateGraph graph1
CreateGraph graph2
CreateGraph graph3
CreateGraph graph4

#Adds nodes
AddNode graph1 SpiderMan
AddNode graph2 SpiderMan
AddNode graph2 Venom
AddNode graph3 SpiderMan
AddNode graph3 Carnage
AddNode graph3 Venom

#Adds edges
AddEdge graph3 SpiderMan Venom SpiderVerse
AddEdge graph3 Venom SpiderMan SpiderVerse

#Finds paths
FindPath graph4 IronMan Hulk

FindPath graph1 SpiderMan Hulk

FindPath graph2 SpiderMan Venom

FindPath graph3 SpiderMan Carnage