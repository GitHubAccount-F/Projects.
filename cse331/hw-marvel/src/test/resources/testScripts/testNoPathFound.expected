#Tests when there is not a path found

#Creates graphs
created graph graph1
created graph graph2
created graph graph3
created graph graph4

#Adds nodes
added node SpiderMan to graph1
added node SpiderMan to graph2
added node Venom to graph2
added node SpiderMan to graph3
added node Carnage to graph3
added node Venom to graph3

#Adds edges
added edge SpiderVerse from SpiderMan to Venom in graph3
added edge SpiderVerse from Venom to SpiderMan in graph3

#Finds paths
unknown: IronMan
unknown: Hulk

unknown: Hulk

path from SpiderMan to Venom:
no path found

path from SpiderMan to Carnage:
no path found